package.json

```diff
@@ -25,6 +25,7 @@
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static --no-version-updates",
     "pre-build": "lerna run --parallel pre-build -- --env production",
     "build": "yarn pre-build && lerna run --parallel build",
+    "watch": "yarn build -- -- --watch",
```

Comment:
Seeing this warning locally: 

```warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.```

Reply:
In the "future version" we'd have only one `--`, but right now we need two, once which yarn ignores (and provides a warning for) and one that gets forwarded.